src/ethereum_test_tools/vm/opcode.py

```diff
@@ -70,7 +70,7 @@ def __eq__(self, other):
         """
         if isinstance(other, OpcodeMacroBase):
             return self._name_ == other._name_
-        return NotImplemented
+        return False
```

Comment:
this breaks the tests

Reply:
See https://github.com/ethereum/execution-spec-tests/pull/457#discussion_r1538971353

Reply:
See https://github.com/ethereum/execution-spec-tests/pull/457#discussion_r1538971353